src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links start with the `#` symbol followed by the id of the target element, unless it's a placeholder link.